tests/scripts/github_docs_comment.py

```diff
@@ -38,6 +49,22 @@ def get_pr_and_build_numbers(target_url):
     return {"pr_number": pr_number, "build_number": build_number}
 
 
+def patch_comment(github, comment_url, body, docs_url):
+    try:
+        github.patch(comment_url, {"body": body})
+    except error.HTTPError as e:
+        logging.exception(f"Failed to patch docs comment {docs_url}: {e}")
+        exit(1)
+
+
+def post_comment(github, url, body, docs_url):
+    try:
+        github.post(url, {"body": body})
+    except error.HTTPError as e:
+        logging.exception(f"Failed to add docs comment {docs_url}: {e}")
+        exit(1)
```

Comment:
what does the error handling add? it seems like it would log the same info as the exception itself being thrown
```suggestion
        github.post(url, {"body": body})
```